src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -2579,22 +2578,19 @@ T BlockchainLMDB::get_and_convert_block_blob_from_height(uint64_t height) const
 
 block BlockchainLMDB::get_block_from_height(uint64_t height) const
 {
-  LOG_PRINT_L3("BlockchainLMDB::" << __func__);
```

Comment:
Quite recently I was debugging something in here and having all these trace-level outputs was quite useful for zeroing in on my bug; I don't like the idea of removing them.

Plus the way these macros work is that, when trace logging isn't enabled, this is just a quick check against the log level that immediately fails and doesn't do anything.